include/mbedtls/ssl.h

```diff
@@ -1424,6 +1424,19 @@ struct mbedtls_ssl_config
                                      *   \c psk is not \c NULL or \c psk_opaque
                                      *   is not \c 0. */
 
+#if defined(MBEDTLS_ZERO_RTT)
+     /*!< Early data indication:
+      *   0  -- MBEDTLS_SSL_EARLY_DATA_DISABLED (for no early data), and
```

Comment:
There's no value in spelling out the definitions of those macros here - they should never be used directly anyway.